content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -325,6 +325,34 @@ Used on: PersistentVolumeClaim
 
 This annotation will be added to dynamic provisioning required PVC.
 
+### volume.kubernetes.io/selected-node
+
+Used on: PersistentVolumeClaim
+
+This annotation is added to a PVC that has been triggered by scheduler toe dynamically provisioned. Its value is the name of the selected node.
+
+### pv.kubernetes.io/bind-completed
+Example: pv.kubernetes.io/bind-completed "true"
+
+Used on: PersistentVolumeClaim
+
+The value for this annotation must be true to take effect. This annotation indicates that {{< glossary_tooltip text="PersistentVolume" term_id="persistent-volume" >}} attached to the pods must be bound and status is completed.
```

Comment:
This annotation indicates that PersistentVolumeClaim was successfully bound to a PersistentVolume? Persistent volumes are not attached to Pods.